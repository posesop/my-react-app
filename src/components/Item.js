import React, { Component } from 'react';

class Item extends Component {
  render() {
    return (
      <div className="Item">
        <p>{this.props.name}</p>
      </div>
    );
  }
}

export default Item;
