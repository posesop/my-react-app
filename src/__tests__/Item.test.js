import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
//npm install --save-dev react-test-renderer
// npm test -- --coverage
import Item from '../components/Item';

it('has the correct title', () => {
  const component = renderer.create(
    <Item name="Soy un objeto" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
