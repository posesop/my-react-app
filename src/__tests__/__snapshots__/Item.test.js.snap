// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has the correct title 1`] = `
<div
  className="Item"
>
  <p>
    Soy un objeto
  </p>
</div>
`;
